### PR TITLE
PD-290: Create PVCs with ReadOnlyMany access mode

### PR DESCRIPTION
**Pull request by andreipope**
Signed-off-by: Andrei Popescu <andrei.popescu@gmail.com>

### CODE REVIEW

**Comment by andreipope** [comment]
@talakad @nathan-px Hey, please review this PR when you get the chance. Thanks!

File: content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+This guide shows how to mount a disk in `ReadOnlyMany`(ROM) access mode. The in-tree Portworx driver for Kubernetes does not support creating PVCs with `ReadOnlyMany(ROM)` access mode. To achieve this functionality, follow the below steps:
+
+1. Create a `sharedv4` volume. Note that you can access a `sharedv4` volume from multiple pods at the same time. For details about creating a `sharedv4` volume, refer to the [Create sharedv4 PVCs](/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-sharedv4-pvcs/) page. 
+
+2. Include a `persistentVolumeClaim` in the `volumes` section of your Pod and set the `readOnly` parameter to `true`.
+
+    Example:
+
+        ```text
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md]: This indentation doesn't appear to render correctly on my local build.

File: content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: Create PVCs using the ReadOnlyMany access mode
+weight: 12
+keywords: Portworx, container, Kubernetes, storage, k8s, PV, persistent disk, PVC, ReadOnlyMany
+description: Learn how to create PVCs using the ReadOnlyMany access mode
+series: k8s-vol
+---
+
+This guide shows how to mount a disk in `ReadOnlyMany`(ROM) access mode. The in-tree Portworx driver for Kubernetes does not support creating PVCs with `ReadOnlyMany(ROM)` access mode. To achieve this functionality, follow the below steps:
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md]:
```suggestion
This guide provides steps for mounting a disk in the `ReadOnlyMany`(ROM) access mode. The in-tree Portworx driver for Kubernetes does not support creating PVCs with the `ReadOnlyMany(ROM)` access mode. To achieve this functionality, follow the steps below:
```

File: content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Create a `sharedv4` volume. Note that you can access a `sharedv4` volume from multiple pods at the same time. For details about creating a `sharedv4` volume, refer to the [Create sharedv4 PVCs](/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-sharedv4-pvcs/) page. 
+
+2. Include a `persistentVolumeClaim` in the `volumes` section of your Pod and set the `readOnly` parameter to `true`.
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md]:
```suggestion
2. Include a `persistentVolumeClaim` in the `volumes` section of your Pod and set the `readOnly` parameter to `true`:
```

File: content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+    Example:
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md]:
```suggestion

```

File: content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md (hunk 3 above)
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md]: Do they link the SharedV4 PVC to the pod here?

File: content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md (hunk 4 above)
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md]:
```suggestion
```

File: content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md (hunk 2 above)
Comment (talakad) [reply content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md]: Remove  "a disk".  instead how about, This guide provides steps for mounting a portworx volume in `ReadOnlyMany` ?

File: content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md (hunk 1 above)
Comment (andreipope) [reply content/portworx-install-with-kubernetes/storage-operations/create-pvcs/create-readonlymany-pvcs.md]: @nathan-px Fixed.

**Comment by andreipope** [comment]
@nathan-px This is updated.

**Pull request merged**